src/commander.c

```diff
@@ -217,15 +213,68 @@ void commander_fill_report(const char *attr, const char *val, int err)
 }
 
 
-void commander_fill_report_signature(const uint8_t *sig, const uint8_t *pubkey)
+static void commander_fill_checkkey(const char *pubkeyhash, int status)
+{
+    char report[40 + 7 + 31] = {0};
```

Comment:
Is there a way to avoid the numbers by `strlen()` or use of constants for PKH size?